webapp/app/forms/steps/lotse/steuerminderungen.py

```diff
@@ -335,7 +335,7 @@ def render(self):
         return render_template(self.template, form=self.render_info.form, render_info=self.render_info,
                                header_title=_('form.lotse.steuerminderungen.header-title'),
                                input_details_title=_('form.lotse.steuerminderungen.details-title'),
-                               input_details_text=_('form.lotse.steuerminderungen.details-text'),
+                               input_details_text=_l('form.lotse.steuerminderungen.details-text'),
```

Comment:
This is exactly that lazy loading change I am talking about

Reply:
:grin: